components/assetcard/stories/template.js

```diff
@@ -52,15 +53,15 @@ export const Template = ({
 			role="figure"
 		>
 			<div class="${rootClass}-assetContainer">
-				<img class="${rootClass}-asset" src="${image ?? exampleImage}" />
+				<img class="${rootClass}-asset" alt="assetcard example image" src="${image ?? exampleImage}" />
 				<div class="${rootClass}-selectionOverlay"></div>
 			</div>
 			${when(
 				title,
 				() => html`<div class="${rootClass}-header">
 					${when(
 						title,
-						() => html`<div class="${rootClass}-title">${title}</div>`
+						() => html`<div class="${rootClass}-title" id=${camelCase(title)}>${title}</div>`
```

Comment:
@pfulton  I'm using `camelCase()` here to use the title as the ID, remove the spacing in multi word titles, and avoid needing to pass a separate ID. I know we're not using Lodash a lot currently. Any concerns about using it here?

Reply:
I think that's fine for now. It shouldn't increase the bundle size/affect performance too much, and it's just for Storybook anyway.

Reply:
Just a heads up @pfulton lodash-es is loaded in storybook for every story already - so not a new dependency